internal/convert/registryv1.go

```diff
@@ -321,6 +321,9 @@ func Convert(in RegistryV1, installNamespace string, targetNamespaces []string)
 	}
 	for _, obj := range in.Others {
 		obj := obj
+		if obj.GetNamespace() == "" {
+			obj.SetNamespace(installNamespace)
+		}
```

Comment:
Based on [this](https://github.com/operator-framework/rukpak/pull/678#issuecomment-1670363234) comment, should we just forcibly set namespace to install namespace?

```suggestion
		obj.SetNamespace(installNamespace)
```

Reply:
If we do so, the only way we allow authors to set namespaces would be through namespace annotation in CSV (as of now). Also, this would also mean that all bundle objects would be deployed in the same NS, different values cannot be configured. I'm fine with it, just want to make sure that we are intentionally making this design decision, and not doing it because of v0, since that behaviour can still be reproduced.